src/schema/v2/system/__tests__/time.test.js

```diff
@@ -45,7 +23,7 @@ describe.skip("SystemTime type", () => {
       }
     `
 
-    const { system } = await runQuery(query, context)
+    const { system } = await runQuery(query, {})
     expect(system.time.day).toEqual(2)
     expect(system.time.iso8601).toEqual("2018-07-02T20:58:58Z")
   })
```

Comment:
These tests are currently _skipped_, presumably because they're difficult to make work consistently across platforms and time zones. I removed the irrelevant stuff but didn't fix the test in this PR.